static/src/javascripts/projects/common/modules/commercial/creatives/template.js

```diff
@@ -54,10 +55,42 @@ define([
         this.params.logosoulmates = svgs('logosoulmates');
         this.params.logosoulmatesjoin = svgs('logosoulmatesjoin');
         this.params.logomembership = svgs('logomembershipwhite');
+        this.params.logosoulmateshorizontal = svgs('logosoulmates');
+        this.params.logomasterclasseshorizontal = svgs('logomasterclasseshorizontal');
+        this.params.logomembershorizontal = svgs('logomembershiphorizontal');
+        this.params.logojobshorizontal = svgs('logojobshorizontal');
+        this.params.logojobs = svgs('logojobs');
+        this.params.logomasterclasses = svgs('logomasterclasses');
         this.params.arrowRight = svgs('arrowRight', ['i-right']);
         this.params.logoguardian = svgs('logoguardian');
         this.params.marque36iconCreativeMarque = svgs('marque36icon', ['creative__marque']);
         this.params.logoFeatureLabel = 'Paid for by';
+
+        this.mapGimbap();
+    };
+
+    Template.prototype.mapGimbap = function () {
```

Comment:
I am not sure about this bit. Any clever ideas how to solve it in more efficient way? @regiskuckaertz